assets/src/edit-story/components/popup/index.js

```diff
@@ -0,0 +1,55 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * External dependencies
+ */
+import { createPortal } from 'react-dom';
+import styled from 'styled-components';
+
+/**
+ * Internal dependencies
+ */
+import { ADMIN_TOOLBAR_HEIGHT } from '../../constants';
+
+const DEFAULT_WIDTH = 270;
+
+const PopUp = styled.div`
+  position: absolute;
+  z-index: 999;
+  top: ${({ y }) => `${y}px`};
```

Comment:
Sometimes it's more convenient to do `right` and `bottom` - it depends on the caller's preference and current screen position. Let's go with the most convenient method for your case and just keep in mind that this may need to be expanded.

Reply:
The `right` value from `getBoundingClientRect` from the `anchor` seems a bit unexpected, I might be missing something of course since I've mostly been using `top` and `left` as a preference.

The `anchorRect.right` value gives me quite a large value which sends the popup to the left edge of the screen. I can look into it again tomorrow to understand why it is as it is, but left it currently using `top` and `left` until then.

Reply:
The `DOMRect.right != css.right`. The `DOMRect.right == DOMRect.left + DOMRect.width`. So it's kind of useless. But `css.right` can be calculated simply as `css.right = bodyRect.right - nodeRect.right`, I think.